client/.prettierignore

```diff
@@ -0,0 +1,3 @@
+#if you like to use prettier extension, please comment out the lines below
+*.tsx
+*.ts
```

Comment:
Is there a reason why we shouldn't use prettier (or equivalent) consistently? I think consistent use of a linter/formatter will result in a much cleaner codebase.